native_client/javascript/package.json.in

```diff
@@ -7,6 +7,17 @@
        "deepspeech": "./client.js"
     },
     "author" : "Mozilla",
+    "license": "MPL-2.0",
+    "homepage": "https://github.com/mozilla/DeepSpeech/tree/$(PROJECT_VERSION)#project-deepspeech",
+    "files": [
```

Comment:
Is this required? Would prefer to not add a list that needs to be updated every time file names/locations change if that's possible.

Reply:
We can remove that, but then we end up packaging a lot of useless elements. Up to you, though :)